recipes/at-spi2-core/all/conanfile.py

```diff
@@ -0,0 +1,90 @@
+from conans import ConanFile, Meson, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class LibnameConan(ConanFile):
+    name = "at-spi2-core"
+    description = "It provides a Service Provider Interface for the Assistive Technologies available on the GNOME platform and a library against which applications can be linked"
+    topics = ("conan", "atk", "accessibility")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://gitlab.gnome.org/GNOME/at-spi2-core/"
+    license = "LGPL-2.1-or-later"
+    generators = "pkg_config"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_x11": [True, False],
+        }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_x11": False,
+        }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+        
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        # FIXME: this package is linux only?
```

Comment:
I don't see any throws on mobile... is this being built on windows?

Reply:
no, because it requires dbus, which is linux/freebsd only. and as far as I know, it does not make sense to compile at-spi2-core without dbus : https://github.com/GNOME/at-spi2-core/blob/3faf8f76135e6b13b2ee871b8375a18a6b918a2d/atspi/meson.build#L82

Reply:
Ah thanks for explaining.

the requires must be blocking platform support?

Reply:
yes, the ConanInvalidConfiguration is transitive

Reply:
Cool, I did not know that!